[*] running method core_negotiate_tlv_encryption
[*] Negotiating TLV encryption
[*] RSA key: 30820122300d06092a864886f70d01010105000382010f003082010a0282010100ca3a6303176a51b29c0b58b46390fe98f2fedf56bbc4b2a879d496b56b585390c92347d812dac3ac28185a8bf640a77b20b1486279120c7edff6f906929f2360556fdcfefb3ad52a124b77c98176090c6a12eaca86473b8652cd9bcdbe7504ba8c1e6d0888550c805ac2624039cadf6f2d4639384e04adab8f094df4114df36f4c7207810e0d9ff6c53e39df420bad3660275fc2de59a57026c9fe45a2863ba8f215c699f863e3b41afc158541e918d50f858abb3f47f15f0143756ec07a34f451bd5b12d1f51ba1fd63f5269c14f973ba62c9fdcf57313c29558eec54271f6608b305c275b1b287ac769474be3d501c7cee25bacb5a69402a2b478289ed61a70203010001
[*] AES key: 0x66c556da848c0f4f140a440d62cba71ef1c54a25ec04bd71e0eb6dbeef20863a
[*] Encrypted AES key: 0x1bb81fadb9c2d0815bc95dfbee37a8970dc6bf50b30bff34a616691ba95b6f5b81a850891e0f4ac8f36a3ef3049a307625b42c692ae067f85c25c40f6c4cd6876027f456ba9481ffd39d119b2b8e1d9dbe1a0794e301ac09cd36d70f902dcd6b60de13695c4e79e8f033759c1a728076c86b469fcfccc2d565afa311c997976afa98d87077011fbedfdb76562a64cfb33ca09768469b74d68616f7705bf044d528d0e39feced4dd3bc1edeb5459559263e08327cd45b13f390f4e65b28928cf2498a070285c6e217177be70a989a3648bbe3a982aa96f726a2fbe7b22c544edf74407ca900a18b75e77aa3ac8c0e9e5b8c63e8fe76991b37c878998b4a78e1f9
[*] TLV encryption sorted
[*] sending response packet
[*] running method core_set_session_guid
[*] sending response packet
[*] running method core_enumextcmd
[*] sending response packet
[*] running method core_enumextcmd
[*] sending response packet
[*] running method core_loadlib
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_sys_config_getuid
[*] sending response packet
[*] running method stdapi_sys_config_sysinfo
[*] sending response packet
[*] running method core_set_uuid
[*] sending response packet
[*] running method stdapi_net_config_get_interfaces
[*] sending response packet
[*] running method stdapi_net_config_get_routes
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_separator
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir-zcxroltk'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 1 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 1
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 2 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 2
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 3 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 3
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 4 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 4
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 5 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 5
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\boot.ini'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\pagefile.sys'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '/etc/master.passwd'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\notepad.exe'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: '%WINDIR%\\system32\\calc.exe'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 6 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 6
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 7 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 7
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 8 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 8
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 9 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 9
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 10 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 10
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 11 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_eof
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 11
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 12 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 12
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-moved'

[*] sending response packet
[*] running method stdapi_fs_file_move
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_file
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'c:\\'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 13 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 13
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir/directory'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 14 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 14
[*] sending response packet
[*] running method stdapi_fs_ls
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'command -v ln || which ln && echo SLGRJMKO']
[*] added process id: 3416, handle: 1
[*] added channel id: 15 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method core_channel_read
[-] method core_channel_read resulted in error: #1
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 1
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 16 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 16
[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir $(pwd)/meterpreter-test-dir.lnk']
[*] added process id: 3421, handle: 2
[*] added channel id: 17 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 2
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.lnk'

[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.1'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[-] method stdapi_fs_stat resulted in an error
Traceback (most recent call last):
  File "<string>", line 1717, in create_response
  File "ext_server_stdapi.py", line 2084, in stdapi_fs_stat
  File "ext_server_stdapi.py", line 891, in get_stat_buffer
FileNotFoundError: [Errno 2] No such file or directory: 'meterpreter-test-dir.2'

[*] sending response packet
[*] running method stdapi_fs_mkdir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method core_channel_open
[*] core_channel_open dispatching to handler: channel_open_stdapi_fs_file
[*] added channel id: 18 type: MeterpreterFile
[*] sending response packet
[*] running method core_channel_write
[*] sending response packet
[*] running method core_channel_close
[*] closed and removed channel id: 18
[*] sending response packet
[*] running method stdapi_sys_process_execute
[*] starting process: ['/bin/sh', '-c', 'ln -s $(pwd)/meterpreter-test-dir.1 $(pwd)/meterpreter-test-dir.2/link']
[*] added process id: 3427, handle: 3
[*] added channel id: 19 type: MeterpreterProcess
[*] sending response packet
[*] running method core_channel_read
[*] sending response packet
[*] running method stdapi_sys_process_close
[*] closed and removed process handle: 3
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_stat
[*] sending response packet
[*] running method stdapi_fs_delete_dir
[*] sending response packet
[*] running method stdapi_sys_config_getenv
[*] sending response packet
[*] running method stdapi_fs_chdir
[*] sending response packet
[*] running method stdapi_fs_getwd
[*] sending response packet
